base/scripts/code.game.cs

```diff
@@ -843,11 +845,13 @@ function remoteSetArmor(%player, %armorType)
 
     Game::clientKilled(%playerId, %killerId);
     
-    zadmin::ActiveMessage::All(KillTrak, %killerId, %playerId, $zadmin::WeaponName[%damageType]);
-
-    //No longer needed?
-    //%killerTeam = Client::GetTeam(%killerId);
-    //%victimTeam = Client::GetTeam(%playerId);
+    //collect stat of how many times player craters into the ground
+    if (%playerCratered) {
+        zadmin::ActiveMessage::All(KillTrak, %killerId, %playerId, "Landing");
+    }
+    else {
+        zadmin::ActiveMessage::All(KillTrak, %killerId, %playerId, $zadmin::WeaponName[%damageType]);
+    }

```

Comment:
can be more concise - no need for the intermediate variable if its not reused:

```suggestion
    if (%damageType == $LandingDamageType) {
        zadmin::ActiveMessage::All(KillTrak, %killerId, %playerId, "Landing");
    }
    else {
        zadmin::ActiveMessage::All(KillTrak, %killerId, %playerId, $zadmin::WeaponName[%damageType]);
    }

```

Reply:
please look below